.github/workflows/main.yaml

```diff
@@ -48,6 +48,9 @@ jobs:
       - name: "Run ESlint check on frontend"
         run: yarn lint
 
+      - name: "Check that TranslationKey type is up to date"
```

Comment:
Maybe 'Generate the TranslationKey type from translation json files' because this is not a check per se.